scripts/rebuild-all.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path')
+const rimraf = require('rimraf')
+const mkdirp = require('mkdirp')
+const cp = require('child_process')
+
+const coreBuildDir = path.resolve(__dirname, '../deltachat-core/builddir')
+console.log(`>> Removing ${coreBuildDir}`)
+rimraf.sync(coreBuildDir)
+
+console.log(`>> Creating ${coreBuildDir}`)
+mkdirp.sync(coreBuildDir)
+
+console.log('>> meson')
+cp.spawnSync('meson', [
```

Comment:
Here we can do more funky stuff later, depending on which platform we are building on, e.g. if we wanted to do something different for windows we can just check `process.platform` etc.